src/components/__tests__/ListCardPost.cy.js

```diff
@@ -147,6 +144,8 @@ function coreTests() {
       .should('be.visible')
       .and('have.css', 'border-color', hexToRgb(gray10))
       .and('have.css', 'border-radius', '28px')
-      .and('contain', button.title);
+      .and('contain', i18n.global.t('index.cardListPost.button'))
+      .invoke('attr', 'href')
+      .should('equal', buttonUrl);
```

Comment:
Add button URL test with request func structure as https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/d1de13093894012841bd1b4c51e486d97e093a4a/src/components/__tests__/FooterBar.cy.js#L238-L246 to prevent test fails due HTTP 429 Too Many Requests error, please.

Reply:
Added test for button link.